azure-pipelines.yml

```diff
@@ -15,6 +15,17 @@ pr:
 variables:
 - template: ${{ variables.Pipeline.Workspace }}/.azure-pipelines/templates/variables.yml
 
+parameters:
+- name: architectures
+  type: object
+  default:
+  - name: AMD64
+    value: amd64
```

Comment:
FYI, Knack previously supported `ppc64le` arch when Travis was used as the build pipeline.